providers/go-feature-flag/src/test/java/dev/openfeature/contrib/providers/gofeatureflag/GoFeatureFlagProviderTest.java

```diff
@@ -514,7 +514,7 @@ void should_publish_events() {
         assertEquals(0, g.getEventsPublisher().publish(), "first attempt expected to fail");
 
         // simulate publish on next interval
-        assertEquals(3, g.getEventsPublisher().publish(), "expected to publish all events after retry");
+        assertEquals(2, g.getEventsPublisher().publish(), "expected to publish all events after retry");
```

Comment:
why ? there are 3 evaluations before this, why expect 2 and not 3 events ?
See also comment above for skipping the event.

Reply:
the first evaluation hit the relay proxy so should not be taken in consideration because the data collection will be done in the relay proxy directly.

We are sending to the endpoint `/data/collector` only the evaluation done by the cache.

Reply:
Cool. It was not clear from the issue, can you possibly update the [requirements](https://gofeatureflag.org/specification/provider-cache) documentation for it?

Reply:
Yes I will update it !

Reply:
Done, waiting for the website to be deployed.